src/System.Data.SqlClient/tests/FunctionalTests/SqlConnectionTest.cs

```diff
@@ -22,6 +24,8 @@ public void ConnectionTest()
             }
         }
 
+
+        [Fact]
```

Comment:
@FransBouma I missed deleting the `[Fact]` attribute while resolving merge conflicts. 
Can you push a change by deleting `[Fact]` line from this test ?